vscode-intellisense/src/completion/utils/isOffsetPossiblyInAClass.test.ts

```diff
@@ -30,13 +30,13 @@ describe("isOffsetPossiblyInAClass", () => {
     test('it recognizes "class" attr', () => {
       const documentText = `<div class=""></div>`;
       const offset = 11;
-      expect(_isStartOfClassValue(documentText, offset)).toBe(true);
+      expect(_isStartOfClassorClassNameValue(documentText, offset)).toBe(true);
```

Comment:
I think we should rename description of above `describe`-block to reflect the new function name and add a test case , that ir recognizes "className" attr.

Would you mind renaming the function to `_isStartOfClassOrClassNameValue`, so it is completely camel case?